molecules/cluster-resources/config/externals/postgresql--idp/values.yaml

```
@@ -0,0 +1,1614 @@
+  ## @param auth.database Name for a custom database to create
+  ##
+  # database: ""
+  ## @param auth.replicationUsername Name of the replication user
+  ##
+  replicationUsername: repl_user
+  ## @param auth.replicationPassword Password for the replication user. Ignored if `auth.existingSecret` is provided
+  ##
+  replicationPassword: ""
+  ## @param auth.existingSecret Name of existing secret to use for PostgreSQL credentials. `auth.postgresPassword`, `auth.password`, and `auth.replicationPassword` will be ignored and picked up from this secret. The secret might also contains the key `ldap-password` if LDAP is enabled. `ldap.bind_password` will be ignored and picked from this secret in this case.
+  ##
+  existingSecret: ""
```

yamllint (1.35.1)

[error] duplication of key "existingSecret" in mapping

(key-duplicates)